guides/documentation_guide.md

```diff
@@ -0,0 +1,28 @@
+# Writing notes for each algorithm
+When creating the notes for algorithms of various categories, please use this template:
+
+```
+# Algorithm Name
+Description
+
+## Explanation
+Pictorial diagram to explain the algorithm 
+
+> Image credits: (name of the site/organisation from where the image was referenced)
+
+## Algorithm
+Algorithm in words
+
+### PseudoCode
```

Comment:
If you go with my previous suggestion, you may want to omit this header. (Suggestion)

Reply:
Yeah, agree. Will do that!